src/Jira/Api/Client/CurlClient.php

```diff
@@ -52,10 +53,10 @@ public function __construct()
      */
     public function sendRequest($method, $url, $data = array(), $endpoint, AuthenticationInterface $credential, $isFile = false, $debug = false)
     {
-        if (!($credential instanceof Basic)) {
-            throw new \Exception(sprintf("CurlClient does not support %s authentication.", get_class($credential)));
+        if (!($credential instanceof Basic) && !($credential instanceof Anonymous)) {
+            throw new \Exception(sprintf("PHPClient does not support %s authentication.", get_class($credential)));
         }
-
+        
```

Comment:
Please revert this change. Also enabling setting in IDE to trim whitespaces on empty lines is a recommended approach.